src/Raven.Client/Documents/Operations/ETL/EtlProcessState.cs

```diff
@@ -9,23 +9,23 @@ public class EtlProcessState : IDatabaseTaskStatus
     {
         public EtlProcessState()
         {
-            LastProcessedEtagPerNode = new Dictionary<string, long>();
+            LastProcessedEtagPerDbId = new Dictionary<string, long>();
             ChangeVector = null;
         }
 
         public string ConfigurationName { get; set; }
 
         public string TransformationName { get; set; }
 
-        public Dictionary<string, long> LastProcessedEtagPerNode { get; set; }
+        public Dictionary<string, long> LastProcessedEtagPerDbId { get; set; }
 
         public string ChangeVector { get; set; }
 
         public string NodeTag { get; set; }
 
-        public long GetLastProcessedEtagForNode(string nodeTag)
+        public long GetLastProcessedEtagForDbId(string dbId)
         {
-            if (LastProcessedEtagPerNode.TryGetValue(nodeTag, out var etag))
+            if (LastProcessedEtagPerDbId.TryGetValue(dbId, out var etag))
```

Comment:
How do we ensure backward compatibility? `LastProcessedEtagPerDbId` property will be empty after upgrading the server so it will start to ETL documents from scratch?